docs/shared/migration/angular-integrated.md

```diff
@@ -0,0 +1,103 @@
+# Migrating an Angular CLI workspace to an Integrated Nx Monorepo
+
+To take advantage of Nx's monorepo features provided by Nx and the Nx Angular plugin, you can also perform a migration from an Angular CLI to an Integrated Nx Monorepo with the command:
+
+```shell
+ng add @nrwl/angular@<version_number>
+```
+
+**Note**: To migrate to legacy versions of Nx prior to Nx 13.10, run the command:
+
+```shell
+ng add @nrwl/workspace@<version_number>
+```
+
+**Note**: Refer to the [Nx and Angular Version Compatibility Matrix](/packages/angular/documents/angular-nx-version-matrix) for matching Angular and Nx versions.
+
+**Note**: Support for workspaces with multiple applications and libraries was added in Nx v14.1.0. If you are migrating using an older version of Nx, your workspace can only contain one application and no libraries in order to use the automated migration, otherwise, you can still [migrate manually](#transitioning-manually).
+
+> The automated migration supports Angular CLI workspaces with a standard structure, configurations and features. If your workspace has deviated from what the Angular CLI generates, you might not be able to use the automated migration and you will need to [manually migrate your workspace](#transitioning-manually).
```

Comment:
Also, this one: should this be a dedicated section about "prerequisites" maybe? I feel like I tend to read over such quotes